processor/tailsamplingprocessor/README.md

```diff
@@ -429,4 +429,83 @@ As a rule of thumb, if you want to add probabilistic sampling and...
 
 **A.** This is likely a load issue. If the collector is processing more traces in-memory than the `num_traces` configuration
 option allows, some will have to be dropped before they can be sampled. Increasing the value of `num_traces` can
-help resolve this error, at the expense of increased memory usage.
\ No newline at end of file
+help resolve this error, at the expense of increased memory usage.
+
+## Monitoring and Tuning 
+
+See [metrics.go][metrics_go] for the full list metrics available for this component.
+
+### Dropped Traces
+
+A circular buffer is used to ensure the number of traces in-memory doesn't exceed `num_traces`. When a new trace arrives, the oldest trace is removed. This can cause a trace to be dropped before it's sampled. To reduce the chance of this happening, either increase `num_traces` or decrease `decision_wait`. Both of those options increase memory usage.
+
+**Number of Traces Dropped**
+```
+otelcol_processor_tail_sampling_sampling_trace_dropped_too_early
+```
+
+**Pre-emptively Preventing Dropped Traces**
+
+A trace is dropped without sampling if it's removed from the circular buffer before `decision_wait`.
+
+To track how long traces remain in the buffer use:
+```
+otelcol_processor_tail_sampling_sampling_trace_removal_age
+```
+
+It may be useful to calculate latency percentiles like p1 and compare that value to `decision_wait`. Values close to `decision_wait` are at risk of being dropped if trace volume increases.
+
+**Slow Sampling Evaluation**
+```
+otelcol_processor_tail_sampling_sampling_decision_timer_latency
+```
+This measures latency of sampling a batch of traces and passing sampled traces through the remainder of the collector pipeline. A latency exceeding 1 second can delay sampling decisions beyond `decision_wait`, increasing the chance of traces being dropped before sampling.
+
+It's therefore recommended to consume this component's output with components that trigger an asynchronous or fast export, like the `batch` processor.
```

Comment:
I wonder if we should name the batch processor specifically, given we know we'll deprecate it in favor of native batching on exporters.

Reply:
good to know, it no longer mentions `batch` processor specifically, just fast/asynchronous consumers in general